tasks/natural-language-processing/text-summarization/Pritam/README.md

```diff
@@ -0,0 +1,27 @@
+input: 
+
+   input text file path: 
+
+       input_text_data_to_summarise.txt
+	   
+	and  
+
+   input number of sentences in summary output: 
+
+       a integer number less than the input number of sentences in text file. 
+
+output: 
+
+   summary_file.txt 
+
+.
+
+command: 
```

Comment:
Please add `mlflow run .` example.

Optional: If possible, please try to implement [serve example](https://mlflow.org/docs/latest/quickstart.html#saving-and-serving-models)